python/lsst/ts/ATWhiteLightSource/chillerComponent.py

```diff
@@ -13,7 +14,7 @@ def __init__(self):
         self.timeout = 5
         self.response_dict = {}
         self.last_response = None
-        self.chiller_com_lock = asyncio.Lock
+        self.chiller_com_lock = asyncio.Lock()
 
     async def connect(self):
         """Connect to chiller's ethernet-to-serial bridge"""
```

Comment:
Is there really a need for `self.connect_task`? 

if seems to me that you could get away with:

```
        if self.connected:
            raise RuntimeError("Already connected")

        self.reader, self.writer = await asyncio.wait_for(asyncio.open_connection(self.ip, self.port), self.timeout)
```

also, clear the print statements... use `self.log.debug` instead...